packages/core/src/client.ts

```diff
@@ -487,7 +489,7 @@ export abstract class Client {
         runAfterNotifyHandlers(merge(notice, {
           id: uuid
         }), this.__afterNotifyHandlers)
-        this.logger.info(`Error report sent ⚡ https://app.honeybadger.io/notice/${uuid}`)
+        this.logger.info(`Error report sent to ⚡ ${endpointUrl}`)
```

Comment:
I don't think this right... we don't want to be displaying the same URL that is used to submit notices to the API. Instead we want to display the corresponding _app_ URL with the `notice/${uuid}` path so that the user can click that link and be redirected to the error detail page in the UI.